lib/services/state/index.ts

```
@@ -87,6 +87,7 @@ class StateManager extends AbstractManager<{ utils: typeof utils }> implements I
     const dataApi = await this.services.dataAPI.get(context.data?.reqHeaders?.authorization);
     const api = new CacheDataAPI(dataApi);
     const version = await api.getVersion(context.versionID!);
+    const project = await api.getProject(version.projectID).catch(() => undefined);
```

why the catch? it feels like an antipattern, why would we be hiding errors here?